types.ts

```diff
@@ -13,6 +13,8 @@ export interface TimeData {
 }
 
 export interface WeatherData {
+    /** The WeatherProvider that generated this data. */
+    weatherProvider: WeatherProviderId;
```

Comment:
Should we add a `timestamp` to this structure as the data might hand around for up-to an hours in App. So we could display a "Data valid @ ...".

Reply:
Right now, the app automatically adds the timestamp generated from the client side at the time the reply is returned. Since we don't use any caching on the API side I don't think there is a difference at this point. Maybe we can extend this when something such as caching is added.

In regards to the app, since we already have that data, we can show `data obtained at`. Where would you suggest we show this? Maybe bottom of the forecast page?

Reply:
Resolving through Issue #47 .